{"name": "test_simple", "status": "failed", "statusDetails": {"message": "AssertionError: assert 4 == 5", "trace": "self = <test_login_page.TestLoginPage object at 0x7f5fb1ca6120>\n\n    def test_simple(self):\n        self.login_page.open()\n        self.login_page.enter_login(\"123\")\n>       assert 4 == 5\nE       assert 4 == 5\n\ntests/test_login_page.py:24: AssertionError"}, "start": 1747852543624, "stop": 1747852544711, "uuid": "813e5d9e-b2db-417c-aa30-cefed30b2704", "historyId": "b625b33df7b710c19b0b63ad85e53788", "testCaseId": "b625b33df7b710c19b0b63ad85e53788", "fullName": "tests.test_login_page.TestLoginPage#test_simple", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_page"}, {"name": "subSuite", "value": "TestLoginPage"}, {"name": "host", "value": "2b93fef25b18"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_page"}]}